Analysis/include/BaseEventAnalyzer.h

```diff
@@ -131,6 +105,7 @@ class BaseEventAnalyzer : public EventAnalyzerCore {
              if(sample.sampleType == SampleType::DY)
                  dymod = std::make_shared<DYModel>(sample,args.working_path());
         }
+        bTagger = std::make_shared<BTagger>(ana_setup.period, ana_setup.jet_ordering);
```

Comment:
bTagger should be initialized in EventAnalyzerCore constructor and shouldn't be shared_ptr